app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageController.scala

```diff
@@ -156,11 +156,18 @@ class SingleDisposalsTriageController @Inject() (
             )
               state.map(_._2)
             else {
-              val newAnswers =
-                answers.fold(
-                  _.copy(disposalMethod = Some(disposalMethod)),
-                  _.copy(disposalMethod = disposalMethod)
-                )
+              val furtherReturn = state.fold(_.isFurtherReturn, _._2.isFurtherReturn).contains(true)
+              val newAnswers    =
+                if (furtherReturn)
```

Comment:
the `if` and `else` branches are the same here - looks like you don't need an `if`-`else`here

Reply:
gainOrLossAfterReliefs needs to be unset below